awx/ui/test/e2e/commands/waitForAngular.js

```diff
@@ -1,5 +1,6 @@
 import { AWX_E2E_TIMEOUT_ASYNC } from '../settings';
 
+/* Post-login utility function that waits for the application to fully load. */
```

Comment:
Is this waiting for AJAX calls? Maybe worth mentioning it?

Reply:
@jakemcdermott might be able to answer that question.